runtime/wasm/src/asc_abi/class.rs

```diff
@@ -429,13 +429,29 @@ pub(crate) struct AscEthereumTransaction {
 
 #[repr(C)]
 #[derive(AscType)]
-pub(crate) struct AscEthereumEvent {
+pub(crate) struct AscEthereumTransactionV2 {
```

Comment:
I wonder if we should make this `AscEthereumTransaction_0_0_2` or something to reflect the spec version that this data structure was introduced in. (The original can remain `AscEthereumTransaction` since it was introduced at the very beginning.) Thoughts?

Reply:
Sounds like a good idea.